src/Staker.sol

```diff
@@ -26,6 +26,13 @@ import {SafeCast} from "openzeppelin/utils/math/SafeCast.sol";
 /// received, the reward duration restarts, and the rate at which rewards are streamed is updated
 /// to include the newly received rewards along with any remaining rewards that have finished
 /// streaming since the last time a reward was received.
+///
+/// The rate at which a depositor earns rewards is proportional to their earning power. Earning
+/// power is based on the amount the depositor has staked and the activity of their delegatee.
+/// The calculation of earning power is handled by a separate module called the earning power
+/// calculator. This module is set by the DAO, and can be updated by the DAO. This gives the DAO
+/// the ability to define and iterate on its own definition of active, aligned participation,
+/// and decide how to reward it.
```

Comment:
Nit: I would replace DAO with admin and give a OZ governor as an example of an acceptable admin.